requests_toolbelt/adapters/ssl.py

```diff
@@ -48,3 +54,7 @@ def init_poolmanager(self, connections, maxsize, block=False):
             maxsize=maxsize,
             block=block,
             ssl_version=self.ssl_version)
+
+    def proxy_manager_for(self, *args, **kwargs):
+        kwargs['ssl_version'] = self.ssl_version
```

Comment:
This method will not cause problems on 2.4.0 because the method it's overriding doesn't exist. =)